src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
-body {
-  margin: 0;
+@import "./utils/_variables";
+@import "./utils/normalize";
+
+@function getShift($cell-size, $gap-size, $value) {
+  @return ($value * $cell-size) + ($value * $gap-size);
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
```

Comment:
make sure that after hover cell should't go outside screen

Reply:
Please advise how to implement without increasing padding or decreasing transform. Thanks

Reply:
add vertical padding more may be of 30px

Reply:
done.